### PR TITLE
treewide: remove deprecated modules

### PR DESCRIPTION
**Pull request by dylad**
### Contribution description
This PR removes a bunch of deprecated modules that I found while looking at the `makefiles/pseudomodules.inc.mk` so I guess it is time to do some cleanup.

There are still some deprecated modules in the wild, but they will need a little bit more attention so I think it is better if this is done in another PR.

### Testing procedure
I think CI should be enough but you could run a quick `grep` to ensure I didn't forget something.

### Issues/PRs references
None.

### CODE REVIEW

**Review by kfessel — COMMENTED** [review]
i see no reason for not removing they are all on the list for more than a release

ping @maribu added most oft them

**Comment by kfessel** [comment]
seems like murdock found a reason :)

File: sys/shell/doc.txt [hunk 1]
```diff
@@ -20,6 +20,6 @@ modules, if it exists.
 
 A few rarely needed shell commands that needs to be used in addition to the
 `shell_commands` and the module providing the C-API. Examples include `nice`,
-`gnrc_udp_cmd`, or `random_cmd`. Consult the documentation of the modules to
+`shell_cmd_gnrc_udp`, or `random_cmd`. Consult the documentation of the modules to
```
Comment (maribu) [inline sys/shell/doc.txt:23]:
```suggestion
`shell_cmd_gnrc_udp`, or `shell_cmd_random`. Consult the documentation of the modules to
```

**Comment by dylad** [comment]
@kfessel I think CI should be happy now. I also squashed at the same time.

**Review by kfessel — APPROVED** [review]
well past their due for removal

**Comment by dylad** [comment]
Thanks for the reviews !

**Pull request merged**